src/Filesystem/README.md

```diff
@@ -0,0 +1,377 @@
+# Filesystem Service
+## Conceptual Summary
+To eliminate security issues like path traversal, we would like to introduce
```

Comment:
This is the Readme, not a FR or a description of the PR. So instead of "we would like to introduce" just write, "we introduced".